tm2/pkg/amino/benchmark_test.go

```diff
@@ -37,12 +44,17 @@ func BenchmarkBinaryPBBindings(b *testing.B) {
 		b.Run(name+":encode:pbbindings", func(b *testing.B) {
 			_benchmarkBinary(b, cdc, rt, "binary_pb", true)
 		})
+
+		// TODO: fix nil pointer error
```

Comment:
Leftover TODO?

Reply:
It was actually failing with a nil pointer